library/src/text/raw.rs

```diff
@@ -237,6 +237,7 @@ impl LocalName for RawElem {
             Lang::POLISH => "Program",
             Lang::RUSSIAN => "Листинг",
             Lang::SLOVENIAN => "Program",
+            Lang::SWEDISH => "Utdrag", // TODO: not easy to translate, utskrift/utdrag?
```

Comment:
An alternative could be to default to "Listing" like in #1353. I'm gonna mark this PR as help needed to maybe get an opinion from another Swedish speaker.

Reply:
I think defaulting to "Listing" makes sense until someone comes up with a better suggestion.

I really tried to do research here and I asked other Swedish friends for opinions.

Listing does not have a direct translation in Swedish. Utdrag/utskrift/program/förteckning could all work under certain circumstances but nothing is as general as listing.